docs/conf.py

```diff
@@ -42,6 +42,7 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "ext.docstrings",
+    "sphinx_autodoc_typehints",  # must be loaded after napoleon modules
```

Comment:
What is a "napoleon" module? Is it one of the ones above?

Reply:
yes, `ext.docstrings` was extending google's modules but this is no longer needed now as I didn't need to add the extra module in the end.